addons/sale/models/sale_order.py

```diff
@@ -1141,7 +1141,7 @@ def _recompute_taxes(self):
     def action_update_prices(self):
         self.ensure_one()
 
-        self._recompute_prices()
+        self.with_context(pricelist_update=1)._recompute_prices()
```

Comment:
```suggestion
        self.with_context(pricelist_update=True)._recompute_prices()
```

`1` is kinda ambiguous, since all you need is a bool, use one ;)